README.md

```diff
@@ -256,6 +256,16 @@ assertContentDescription(R.id.anyView, R.string.content_description);
 assertContentDescription(R.id.anyView, "Some text");
 ```
 
+#### Is this List empty? How many items does it have?
+````java
+// Works with both ListView and RecyclerView
+assertListNotEmpty(R.id.list)
+assertListItemCount(R.id.list, 5)
+
+// you can also pass custom assertion
```

Comment:
I think it's a typo. It should be `custom assertionS` or `a custom assertion`. I prefer the plural because it's used in the rest of the readme.

Reply:
Oh! And start the phrase with an uppercase 👯‍♂️

Reply:
```suggestion
// You can also pass a custom assertion
```

Reply:
Done! I've used the plural, as @rocboronat says it's used in the rest of the readme.